playground/src/components/generic-list.usage.tsx

```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 import { IUser } from '@src/models';
 import { GenericList } from '@src/components';
 
-export const UserList = class extends GenericList<IUser> { };
+export class UserList extends GenericList<IUser> { };
```

Comment:
linter screaming on the semicolon after class, shouldn't be there
`yarn lint` in playground folder